config/ui-extensions/eventing/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+configMapGenerator:
+  - name: operator.kyma-project.io
```

Comment:
This name needs to be changed

Reply:
yes, it needs to be `eventing.operator.kyma-project.io`.